nb-configuration.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The nb-configuration.xml file should be in your global gitignore so it does not go into version control.